types/index.d.ts

```diff
@@ -48,7 +40,7 @@ declare module 'found' {
     /**
      * map version of search string
      */
-    query: ObjectStringMap;
+    query: Record<string, string | undefined>;
```

Comment:
`null` is allowed here

Reply:
Rather, in a `LocationDescriptorObject`, both `null` and `undefined` are allowed. In a `Location` object, the values are always `string`s.